applications/GeoMechanicsApplication/python_scripts/geomechanics_Pw_solver.py

```diff
@@ -75,15 +75,14 @@ def GetDefaultParameters(cls):
             "block_builder"              : true,
             "search_neighbours_step"     : false,
             "linear_solver_settings":{
-                "solver_type": "AMGCL",
-                "tolerance": 1.0e-6,
-                "max_iteration": 100,
-                "scaling": false,
-                "verbosity": 0,
-                "preconditioner_type": "ILU0Preconditioner",
-                "smoother_type": "ilu0",
-                "krylov_type": "gmres",
-                "coarsening_type": "aggregation"
+                "solver_type":     "amgcl",
+                "smoother_type":   "ilu0",
+                "krylov_type":     "gmres",
+                "coarsening_type": "aggregation",
+                "max_iteration":   100,
+                "verbosity":       0,
+                "tolerance":       1.0e-6,
+                "scaling":         false
```

Comment:
Of course we can discuss if these are the best defaults we can think of, I just took the settings of one of our tests. I first tried sparse_lu, but I don't think it's always guaranteed that the LinearSolversApplication is initialized, so not sure if that's the most robust option for a default